lib/generators/rspec/controller/templates/request_spec.rb

```diff
@@ -1,7 +1,14 @@
 require 'rails_helper'
 
 RSpec.describe "<%= class_name.pluralize %>", <%= type_metatag(:request) %> do
-<% namespaced_path = regular_class_path.join('/') %>
+<% namespaced_path = regular_class_path.join('/') -%>
+<% if actions.empty? -%>
+  describe "GET /index" do
+    it "returns http success" do
```

Comment:
Not specifically related to this pull request.

I'm slightly opposed to adding "returns http success" in request specs. From my point of view, they should test more than just http statuses, and providing this as a reference example in the template is promoting a dubious practice.

Reply:
Indeed. Maybe I can update the template for that as well? Do you have an idea regarding what we can provide? 

I'm thinking to `exists` or `is reachable` (I have a preference for the second option)

Reply:
Maybe `"returns a list of #{class_name.pluralize.downcase}"`? At least this is what an "index" originally means.

Reply:
> update the template for that as well

That would be very helpful!